domains/transfer-agreements/TransferAgreements.API/API/Data/ApplicationDbContext.cs

```diff
@@ -12,4 +12,5 @@ public ApplicationDbContext(DbContextOptions<ApplicationDbContext> options) : ba
 
     public DbSet<TransferAgreement> TransferAgreements { get; set; }
     public DbSet<TransferAgreementHistoryEntry> TransferAgreementHistoryEntries { get; set; }
+    public DbSet<Connection> Connections { get; set; }
```

Comment:
Have you considered using database schemas to mark the difference between Connections and TA?

Reply:
No - I'm unsure what that would add?